### PR TITLE
styled message

### PR DESCRIPTION
**Pull request by lilithkarapetyan**

### CODE REVIEW

**Comment by louisahar** [comment]
Fine

**Review by robhov97 — APPROVED** [review]

File: style.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+p {
+    color: magenta;
+    background-color: black;
+}
```
Comment (robhov97) [inline style.css]: new line please

File: style.css [hunk 2]
```diff
@@ -0,0 +1,4 @@
+p {
+    color: magenta;
```
Comment (lilithkarapetyan) [inline style.css:2]: please change the color

File: index.html [hunk 3]
```diff
@@ -7,6 +7,6 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    Hello my world!!!
+    <p>Hello my world!!!</p>
 </body>
 </html>
```
Comment (robhov97) [inline index.html]: new line please

**Review by robhov97 — CHANGES_REQUESTED** [review]

File: style.css (hunk 1 above)
Comment (lilithkarapetyan) [reply style.css]: done

**Review by robhov97 — APPROVED** [review]
LGTM!

**Pull request merged**